roads.mss

```diff
@@ -2395,4 +2387,4 @@
   [zoom >= 18] {
     line-width: 14;
   }
-}
+}/**/
```

Comment:
Extra comments?

Reply:
a comment trick to comment everything by just adding `/*` at the top